### ISSUE TITLE
Default if importBugSigDB does no import the latest Zenodo stable version

### ISSUE THREAD

**Issue by sdgamboa** [waldronlab/bugsigdbr#57]
From the documentation: "If not specified otherwise, the importBugSigDB function will obtain the most recent stable release from Zenodo by default."

However, I got version 1.1.0 by default with the date Apr 17, 2022 (default value in version argument). I think the most recent stable version is the one released on Dec 19, 2023 (v1.2.0). I could tell because of the presence of the "BSDB ID" column in the latest. Reprex below.

```
library(bugsigdbr)
default <- importBugSigDB(cache = FALSE) # v1.1.0 Apr 17, 2022 "10.5281/zenodo.6468009"
latest <- importBugSigDB(version = "10.5281/zenodo.10407666", cache = FALSE) # v1.2.0 Dec 19, 2023

"BSDB ID" %in% colnames(default)
#> [1] FALSE
"BSDB ID" %in% colnames(latest)
#> [1] TRUE

sessioninfo::session_info()
#> ─ Session info ───────────────────────────────────────────────────────────────
#>  setting  value
#>  version  R version 4.3.2 (2023-10-31)
#>  os       Ubuntu 22.04.3 LTS
#>  system   x86_64, linux-gnu
#>  ui       X11
#>  language (EN)
#>  collate  en_US.UTF-8
#>  ctype    en_US.UTF-8
#>  tz       America/New_York
#>  date     2024-01-31
#>  pandoc   3.1.11 @ /usr/bin/ (via rmarkdown)
#> 
#> ─ Packages ───────────────────────────────────────────────────────────────────
#>  package       * version date (UTC) lib source
#>  BiocFileCache   2.10.1  2023-10-26 [1] Bioconductor
#>  bit             4.0.5   2022-11-15 [1] CRAN (R 4.2.2)
#>  bit64           4.0.5   2020-08-30 [1] CRAN (R 4.1.2)
#>  blob            1.2.4   2023-03-17 [1] CRAN (R 4.2.2)
#>  bugsigdbr     * 1.8.1   2023-10-29 [1] Bioconductor
#>  cachem          1.0.8   2023-05-01 [1] CRAN (R 4.2.2)
#>  cli             3.6.2   2023-12-11 [1] CRAN (R 4.3.2)
#>  curl            5.2.0   2023-12-08 [1] CRAN (R 4.3.2)
#>  DBI             1.2.1   2024-01-12 [1] RSPM (R 4.3.0)
#>  dbplyr          2.4.0   2023-10-26 [1] CRAN (R 4.3.1)
#>  digest          0.6.34  2024-01-11 [1] CRAN (R 4.3.2)
#>  dplyr           1.1.4   2023-11-17 [1] CRAN (R 4.3.2)
#>  evaluate        0.23    2023-11-01 [1] CRAN (R 4.3.1)
#>  fansi           1.0.6   2023-12-08 [1] CRAN (R 4.3.2)
#>  fastmap         1.1.1   2023-02-24 [1] CRAN (R 4.2.2)
#>  filelock        1.0.3   2023-12-11 [1] CRAN (R 4.3.2)
#>  fs              1.6.3   2023-07-20 [1] CRAN (R 4.3.1)
#>  generics        0.1.3   2022-07-05 [1] CRAN (R 4.2.1)
#>  glue            1.7.0   2024-01-09 [1] CRAN (R 4.3.2)
#>  htmltools       0.5.7   2023-11-03 [1] CRAN (R 4.3.2)
#>  httr            1.4.7   2023-08-15 [1] CRAN (R 4.3.1)
#>  knitr           1.45    2023-10-30 [1] CRAN (R 4.3.1)
#>  lifecycle       1.0.4   2023-11-07 [1] CRAN (R 4.3.2)
#>  magrittr        2.0.3   2022-03-30 [1] CRAN (R 4.1.3)
#>  memoise         2.0.1   2021-11-26 [1] CRAN (R 4.1.2)
#>  pillar          1.9.0   2023-03-22 [1] CRAN (R 4.2.2)
#>  pkgconfig       2.0.3   2019-09-22 [1] CRAN (R 4.1.2)
#>  purrr           1.0.2   2023-08-10 [1] CRAN (R 4.3.1)
#>  R.cache         0.16.0  2022-07-21 [1] CRAN (R 4.2.1)
#>  R.methodsS3     1.8.2   2022-06-13 [1] CRAN (R 4.2.0)
#>  R.oo            1.26.0  2024-01-24 [1] CRAN (R 4.3.2)
#>  R.utils         2.12.3  2023-11-18 [1] CRAN (R 4.3.2)
#>  R6              2.5.1   2021-08-19 [1] CRAN (R 4.1.2)
#>  reprex          2.1.0   2024-01-11 [1] CRAN (R 4.3.2)
#>  rlang           1.1.3   2024-01-10 [1] CRAN (R 4.3.2)
#>  rmarkdown       2.25    2023-09-18 [1] CRAN (R 4.3.1)
#>  RSQLite         2.3.5   2024-01-21 [1] CRAN (R 4.3.2)
#>  rstudioapi      0.15.0  2023-07-07 [1] CRAN (R 4.3.1)
#>  sessioninfo     1.2.2   2021-12-06 [1] CRAN (R 4.1.2)
#>  styler          1.10.2  2023-08-29 [1] CRAN (R 4.3.1)
#>  tibble          3.2.1   2023-03-20 [1] CRAN (R 4.2.2)
#>  tidyselect      1.2.0   2022-10-10 [1] CRAN (R 4.2.1)
#>  utf8            1.2.4   2023-10-22 [1] CRAN (R 4.3.1)
#>  vctrs           0.6.5   2023-12-01 [1] CRAN (R 4.3.2)
#>  withr           3.0.0   2024-01-16 [1] CRAN (R 4.3.2)
#>  xfun            0.41    2023-11-01 [1] CRAN (R 4.3.1)
#>  yaml            2.3.8   2023-12-11 [1] CRAN (R 4.3.2)
#> 
#>  [1] /mnt/STORE1/lib/R/bioc-release
#>  [2] /usr/local/lib/R/library
#> 
#> ──────────────────────────────────────────────────────────────────────────────
```

<sup>Created on 2024-01-31 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

**Comment by sdgamboa** [comment]
Or will it be updated in the code in the next Bioc Release?

**Comment by lgeistlinger** [comment]
We are still working on the new release in https://github.com/waldronlab/bugsigdbr/issues/49, should be out soon ...

**Comment by lgeistlinger** [comment]
Resolved via [ba3bc9c](https://github.com/waldronlab/bugsigdbr/commit/ba3bc9c03ac81a9654952573e8c3546a2b11912c); latest zenodo release is obtained in bugsigdbr v1.8.4 (release) and bugsigdbr v1.9.4 (devel).
